torch/testing/_internal/common_methods_invocations.py

```diff
@@ -3438,6 +3465,33 @@ def generator():
 
     return list(generator())
 
+def sample_inputs_conv_transpose3d(op_info, device, dtype, requires_grad, **kwargs):
+    make_arg = partial(make_tensor, device=device, dtype=dtype, requires_grad=requires_grad)
+
+    # Ordered as shapes for input, weight, bias
+    # and a dict of values of (stride, padding, output_padding, groups, dilation)
+    cases: Tuple[Tuple[int], Tuple[int], Tuple[int], dict] = (  # type: ignore[assignment]
```

Comment:
We should update this file so it doesn't type check because the type checking is a pain